api/v1/views/index.py

```diff
@@ -7,7 +7,7 @@
 from api.v1.views import app_views
 
 
-@app_views.route("/status/", methods=['GET'])
```

Comment:
Does the strict_slashes only pertain to the initial slash or why was this changed? Thanks!